File: Domovoy/ClientApp/src/api/models/SerachFilters.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ApartmentState } from './ApartmentState';
import type { ApartmentType } from './ApartmentType';

export type SerachFilters = {
    city?: string | null;
    apartmentType?: ApartmentType;
    apartmentState?: ApartmentState;
    costMin?: number;
    costMax?: number;
    roomCountMin?: number;
    roomCountMax?: number;
};

File: Domovoy/Models/SerachFilters.cs
namespace Domovoy.Models;

public class SerachFilters
{
    public string City { get; set; }
    public ApartmentType? ApartmentType { get; set; }
    public ApartmentState? ApartmentState { get; set; }
    public int CostMin { get; set; }
    public int CostMax { get; set; }
    public int RoomCountMin { get; set; }
    public int RoomCountMax { get; set; }
}